From: Isaac Hermida <isaac.hermida@digi.com>
Date: Fri, 30 Sep 2016 12:46:17 +0200
Subject: [PATCH] hciattach_rome: do not override the baudrate in the NVM file

Do not use hcitattach to override the NMV baudrate because we are
setting it directly in the firmware file.

https://jira.digi.com/browse/DEL-3057

Signed-off-by: Isaac Hermida <isaac.hermida@digi.com>
---
 tools/hciattach_rome.c | 11 ++---------
 1 file changed, 2 insertions(+), 9 deletions(-)

diff --git a/tools/hciattach_rome.c b/tools/hciattach_rome.c
index 59bdc16e4e8f..4f0f8c82fa8d 100644
--- a/tools/hciattach_rome.c
+++ b/tools/hciattach_rome.c
@@ -1903,15 +1903,8 @@ download:
             }
             fprintf(stderr, "%s: Download TLV file successfully \n", __FUNCTION__);
 
-            /*
-             * Overriding the baud rate value in NVM file with the user
-             * requested baud rate, since default baud rate in NVM file is 3M.
-             */
-            err = rome_set_baudrate_req(fd, local_baud_rate, controller_baud_rate);
-            if (err < 0) {
-                fprintf(stderr, "%s: Baud rate change failed!\n", __FUNCTION__);
-                goto error;
-            }
+            /* Do not override the baudrate in the TLV file */
+            fprintf(stderr, "%s: Skipping Override of baud rate!\n", __FUNCTION__);
 
             /* Perform HCI reset here*/
             err = rome_hci_reset_req(fd, local_baud_rate);
